2.x/Dockerfile

```diff
@@ -90,48 +85,20 @@ RUN set -x \
     && git -C /usr/src/tarantool submodule update --init --recursive \
     && (cd /usr/src/tarantool; \
        cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo\
-             -DENABLE_BUNDLED_LIBYAML:BOOL=OFF\
+             -DENABLE_BUNDLED_LIBYAML:BOOL=ON\
+             -DENABLE_BUNDLED_LIBCURL:BOOL=ON\
```

Comment:
Those options are ON by default, so I would not add them explicitly.

Reply:
Fixed